tests/Functional/ContentCallForTendersRenderTest.php

```diff
@@ -79,163 +65,240 @@ public function testTenderRendering(): void {
     ];
-    foreach ($values_data as $index => $value) {
-      $this->assertEquals($value, $values[$index]->getText());
-    }
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
 
-    // Assert header of second field group.
-    $this->assertContentHeader($content_items[1], 'Description', 'description');
+    // Assert "Open" status.
+    $node->set('oe_call_tenders_opening_date', ['value' => $opening_date->format('Y-m-d')])->save();
+    $this->drupalGet($node->toUrl());
 
-    // Assert content of second field group.
-    $content_second_group = $content_items[1]->find('css', '.ecl-editor p');
-    $this->assertEquals('Call for tenders body', $content_second_group->getText());
+    $page_header_expected_values['meta'] = 'Call for tenders | Open';
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
 
-    // Assert header of third field group.
-    $this->assertContentHeader($content_items[2], 'Documents', 'documents');
-    $this->assertMediaDocumentDefaultRender($content_items['2'], 'call_for_tenders_document');
+    $details_expected_values['items'] = [
+      [
+        'label' => 'Status',
+        'body' => 'Open',
+      ], [
+        'label' => 'Publication date',
+        'body' => '12 February 2020',
+      ], [
+        'label' => 'Opening date',
+        'body' => '14 February 2020',
+      ], [
+        'label' => 'Deadline date',
+        'body' => '21 February 2020, 01:00 (AEDT)',
+      ],
+    ];
+    $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
+    $this->assertDeadlineDateStrike($content);
 
-    // Assert Responsible department field label.
+    // Assert "Upcoming" status.
+    $opening_date = (clone $static_time)->modify('+ 2 days');
+    $node->set('oe_call_tenders_opening_date', ['value' => $opening_date->format('Y-m-d')])->save();
+    $this->drupalGet($node->toUrl());
+
+    $page_header_expected_values['meta'] = 'Call for tenders | Upcoming';
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
+
+    $details_expected_values['items'] = [
+      [
+        'label' => 'Status',
+        'body' => 'Upcoming',
+      ], [
+        'label' => 'Publication date',
+        'body' => '12 February 2020',
+      ], [
+        'label' => 'Opening date',
+        'body' => '19 February 2020',
+      ], [
+        'label' => 'Deadline date',
+        'body' => '21 February 2020, 01:00 (AEDT)',
+      ],
+    ];
+    $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
+    $this->assertDeadlineDateStrike($content);
+
+    // Assert "Closed" status.
+    $deadline_date = (clone $static_time)->modify('- 2 days');
+    $node->set('oe_call_tenders_deadline', $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT))->save();
+    $this->drupalGet($node->toUrl());
```

Comment:
Same as for events, we don't have to change the date and refresh the page, we need to move the time, run cron and assert the change, or else we are not testing our time based tags.

Please also change this on ContentCallForProposalRenderTest.php